editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -296,6 +307,39 @@ export const AnalyticsProvider: React.FC<{ children: React.ReactNode }> = ({
     }
   }
 
+  async function trackBackwardsNavigationByNodeId(
+    nodeId: string,
+    initiator: BackwardsNaviagtionInitiatorType,
+  ) {
+    const targetNodeMetadata = getTitleAndTypeFromFlow(nodeId);
+    console.log("Target node info: ", targetNodeMetadata);
+    const metadata: Record<string, NodeMetadata> = {};
```

Comment:
This seems like the right call to me! Since you can only click "change" once, and always have to step forward again one by one after changing (and could theoretically drop off before you're back at Review/Result), individual records make sense here to me :+1: